src/components/pages/Mypage/utils/getProfilePhoto.js

```diff
@@ -0,0 +1,8 @@
+import { PROFILE_LIST } from '@common/constants';
+
+const getProfilePhoto = id => {
+  const profile = PROFILE_LIST[id];
+  return profile ?? PROFILE_LIST[8];
+};
```

Comment:
ts로 바꾸고 호깃 id: number 로 타입을 명시해놓는게 어떨까요?
```suggestion
const getProfilePhoto = (id: number) => {
  const profile = PROFILE_LIST[id];
  return profile ?? PROFILE_LIST[8];
};
```

Reply:
PROFILE_LIST[id] 가 undefined 가 되는게 어색한거같습니다.
지금은 객체라서 가능한데, 배열로 바꾸게 된다면 id 와 PROFILE_LIST.length를 비교해서 처리해야할거같습니다

Reply:
이부분 타스로 아직 안바꿔놔서 이런 불상사가 터졌네요 최우선적으로 바꿔보도록 하겠습니다

Reply:
객체를 배열로 바꿨더니 해당 함수가 쓸모가...없어져서 날렸습니다 ㅎ